frontend/src/components/HomePage/ContainerStoryRules/ContainerStoryRules.jsx

```diff
@@ -1,9 +1,46 @@
 import { Link } from "react-router-dom";
+import { useGlobalContext } from "../../Context/GlobalContextProvider";
 import Rules from "./Rules/Rules";
 import Story from "./Story/Story";
 import "./ContainerStoryRules.scss";
 
 function ContainerStoryRules() {
+  const objectivesItemArray = [
+    {
+      objective: "Récupérer l'oeuf du quetzalcoatlus",
+      objectiveClass: "eggObjective",
+      item: "egg",
+    },
+    {
+      objective: "Récupérer la dent du Tyranossaurus Rex",
+      objectiveClass: "toothObjective",
+      item: "tooth",
+    },
+    {
+      objective: "Récupérer une plume de vélociraptor",
+      objectiveClass: "featherObjective",
+      item: "feather",
+    },
+  ];
+
+  const { setObjectives, setBag } = useGlobalContext();
+  function randomObjectives() {
+    const objectivesArrayRandom = [];
+
+    while (objectivesArrayRandom.length < 2) {
```

Comment:
Beware of ninja code: hard-coded values aren't ideal, because they're liable to be forgotten or misunderstood as things evolve.

Reply:
fixed